client/my-sites/plans-single-products/index.jsx

```diff
@@ -2,33 +2,103 @@
  * External dependencies
  */
 import React from 'react';
-import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
 import { isEnabled } from 'config';
 import Product from './product';
+import { getCurrentUserCurrencyCode } from 'state/current-user/selectors';
 
 /**
  * Style dependencies
  */
 import './style.scss';
 
-const PlansSingleProducts = ( { translate } ) => {
+const PlansSingleProducts = props => {
+	const {
+		billingTimeFrame,
+		currencyCode,
+		isPlaceholder,
+		onProductSelect,
+		productProperties,
+	} = props;
 	const displayBackup = isEnabled( 'plans/jetpack-backup' );
 	const displayScan = isEnabled( 'plans/jetpack-scan' );
 
-	if ( ! displayBackup && ! displayScan ) {
-		return null;
-	}
-
 	return (
 		<div className="plans-single-products">
-			{ displayScan && <Product heading={ translate( 'Jetpack Scan' ) } /> }
-			{ displayBackup && <Product heading={ translate( 'Jetpack Backup' ) } /> }
+			{ displayScan && (
+				<Product
+					billingTimeFrame={ billingTimeFrame }
+					currencyCode={ currencyCode }
+					isPlaceholder={ isPlaceholder }
+					onSelect={ onProductSelect }
+					{ ...productProperties.jetpackScan }
+				/>
+			) }
+			{ displayBackup && (
+				<Product
+					billingTimeFrame={ billingTimeFrame }
+					currencyCode={ currencyCode }
+					isPlaceholder={ isPlaceholder }
+					onSelect={ onProductSelect }
+					{ ...productProperties.jetpackBackup }
+				/>
+			) }
 		</div>
 	);
 };
 
-export default localize( PlansSingleProducts );
+export default connect(
+	state => {
+		return {
+			billingTimeFrame: 'per year',
+			currencyCode: getCurrentUserCurrencyCode( state ),
+			isPlaceholder: false,
```

Comment:
I'd always suggest adding `TODO` comments whenever we're building something that we'll need to change for sure in the process of development. Without those comments, it's easy to forget about and leave like this forever.